crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -1959,11 +1958,19 @@ void OnDisable()
 
         public override void OnInspectorGUI()
         {
+            var target = this.target as OceanRenderer;
+
+            if (target._isPrefabStageInstance)
+            {
+                EditorGUILayout.Space();
+                EditorGUILayout.HelpBox("Crest does not support prefab mode. Changes made in prefab mode will not be reflected in the scene view until saved.", MessageType.Warning);
```

Comment:
could you maybe expand "until saved" slightly. does this mean the user has to `File/Save` and it will update the water? perhaps saying directly "save this prefab to update the water" or something?

Reply:
Correct. Sounds good. Done. I also refactored and placed constant string in common area.